scilpy/tracking/tracking_field.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you use the same structure of documentation as in local_tracking

Reply:
Done in 4b2c08d